cookbooks/mediawiki/recipes/default.rb

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 #
 
+include_recipe "apt"
+include_recipe "accounts"
+include_recipe "munin"
```

Comment:
As I said earlier, this is probably not the best way to do it - there are certain recipes are automatically set on all hosts so we don't include them in every cookbook like this. Please, please, look at how the other tests are doing this.

Reply:
Hm, this looked to be a pattern in use in other tests. I see a mix of approaches: some tests have additional recipes in their run list while others include recipes as I’m doing here. Can you point to a specific pattern I should follow?

Reply:
Look at the `otrs` test for example - it runs extra recipes directly in the `.kitchen.yml` definition.

As a general rule if you're having to alter the cookbooks to test them then there's probably something wrong somewhere...

Reply:
Thanks, I’ll dig into `otrs`!